app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,31 @@
  *= require foundation_and_overrides
 
  */
+
```

Comment:
refactor to have one class named flash and 2 other classes (error and success) since they share characteristics